includes/datavalues/SMW_DV_WikiPage.php

```diff
@@ -672,6 +672,13 @@ protected function getShortCaptionText() {
 			$displayTitle = $this->getText();
 		}
 
+		# Remove (context) as per pipe trick.
+		if ( $this->getOption( 'smwgDVFeatures' ) & SMW_DV_WPV_PIPETRICK != 0) {
```

Comment:
Using `if ( $this->hasFeature( 'SMW_DV_WPV_PIPETRICK' ) ) ...` should be more expressive.

Reply:
This patch is forwarde from our 2.4 install so it isn't using the 3.0 features.